runners/direct-java/pom.xml

```diff
@@ -361,6 +361,12 @@
       </exclusions>
       <scope>test</scope>
     </dependency>
-
+    
+    <!-- needed for eclipse-jdt generated core as the test-jar references classes from this -->
```

Comment:
It seems that unit tests from `java-sdk-core` are being run as part of the integration test run. There tests shouldn't be run in the integration suite and then we don't need this dependency.

In maven, we configure surefire to select integration tests with the `groups` filter in the `pom.xml`:

```xml
<groups>org.apache.beam.sdk.testing.NeedsRunner</groups>
```

(I've verified that we see this same error under maven when the groups filter is removed. /cc @lukecwik who helped me investigate)

How are you running integration tests within Eclipse? I wonder why it's not respecting the groups filter.